.storybook/config.js

```diff
@@ -3,7 +3,7 @@ import { setOptions } from "@storybook/addon-options";
 const req = require.context("../elements", true, /\.story\.js$/);
 
 setOptions({
-  name: "RHElements",
+  name: "PFElements",
```

Comment:
PF Elements or PFElements? Space or no-space?

Reply:
It should be PatternFly Elements.

Reply:
I fix it.